### PR TITLE
[BACKLOG-6287] Review unit tests and update documentation for …

### PR DESCRIPTION
**Pull request by davidmsantos90**
…/pentaho/util/MessageBundle.js

Renamed MessageBundleSpec to MessageBundle.Spec to keep it consistent with the rest of the tests we have been doing.

@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by davidmsantos90** [comment]
@dcleao updated PR with your reviews

File: package-res/resources/web/pentaho/util/MessageBundle.js [hunk 1]
```diff
@@ -58,6 +58,17 @@ define([
    * @alias get
    * @memberOf pentaho.util.MessageBundle#
    *
+   * @example
+   * define(["pentaho/util/MessageBundle"], function(MessageBundle) {
+   *    var bundle = new MessageBundle({key1: "value1", key2: "value2_{0}"});
```
Comment (dcleao) [inline package-res/resources/web/pentaho/util/MessageBundle.js]: Cool! I would be even happier if you had a key/get with an object :-)

File: package-res/resources/web/pentaho/util/MessageBundle.js [hunk 2]
```diff
@@ -82,7 +93,28 @@ define([
   };
 
   /**
-   * Object representation of the message bundle.
+   * Gets the hierarchical object representation of the message bundle,
+   * formed from splitting dotted message keys.
+   *
+   * @example
+   * define(["pentaho/util/MessageBundle"], function(MessageBundle) {
+   *    var bundle = new MessageBundle({
```
Comment (dcleao) [inline package-res/resources/web/pentaho/util/MessageBundle.js]: Cool!

File: package-res/resources/web/pentaho/util/MessageBundle.js [hunk 3]
```diff
@@ -112,6 +144,15 @@ define([
    * When a property tag results in a _nully_ value (like when `scope` is not specified),
    * it is replaced by the special marker `"[?]"`.
    *
+   * @example
```
Comment (dcleao) [inline package-res/resources/web/pentaho/util/MessageBundle.js]: Cool!

**Comment by dcleao** [comment]
Cool! Is it straightforward to make that small addition?

**Comment by davidmsantos90** [comment]
Yes, will do it in a sec and ping you again

**Comment by davidmsantos90** [comment]
@dcleao key/get example updated

**Pull request merged**